Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [package]
 name = "simple-cache-rs"
 description = "A simple rust implementation of HashMap with expiration control."
-version = "0.1.0"
-authors = ["Calvin Brown <cab025@protonmail.com>"]
+version = "0.2.0"
+authors = ["Giuseppe Eletto <peppe.eletto@gmail.com>", "Calvin Brown <cab025@protonmail.com>"]
 edition = "2018"
-repository = "https://github.com/calvinbrown085/simple-cache-rs"
+repository = "https://github.com/giuseppe998e/simple-cache-rs"
```

Comment:
Can we change this back? Since the repository isn't changing.

Reply:
No problem, your repository, your rules.

Reply:
Same as below, can you update these in the PR? I don't think I can update them here.